<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HUDMainWindowClass</class>
 <widget class="QMainWindow" name="HUDMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1153</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HUDMainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QToolButton" name="nav_hfs">
        <property name="text">
         <string>HFS</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="nav_log">
        <property name="text">
         <string>LOG</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="nav_scene">
        <property name="text">
         <string>SCENE</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="nav_qml">
        <property name="text">
         <string>QML</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="nav_screensaver">
        <property name="text">
         <string>SCREENSAVER</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="mainstack">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="hfsPage"/>
      <widget class="QWidget" name="logPage">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTreeView" name="hfsTreeView"/>
        </item>
        <item>
         <widget class="QTextBrowser" name="hudlog"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="scenePage"/>
      <widget class="QWidget" name="screensaverPage"/>
      <widget class="QWidget" name="qmlPage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>QML</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="qmllayout"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
